back/engines/commercial/granular_permissions/spec/acceptance/permissions_spec.rb

```diff
@@ -273,7 +273,6 @@
 
         @user.reset_confirmation_and_counts
         @user.update!(
-          email: 'my@email.com',
```

Comment:
Why was the email removed?

Reply:
Wasn't needed by the test and this was failing because of the validation rule I had put in place to prevent email being updated directly